src/libs/UserUtils.js

```diff
@@ -89,7 +89,7 @@ function getDefaultAvatar(accountID = -1) {
 
     // There are 24 possible default avatars, so we choose which one this user has based
     // on a simple hash of their login. Note that Avatar count starts at 1.
```

Comment:
Lets also update the comment as its no longer being hashed

Reply:
done